omod/src/main/webapp/pages/login.gsp

```diff
@@ -21,6 +21,15 @@
     var OPENMRS_CONTEXT_PATH = '${ ui.contextPath() }';
 </script>
 
+<script type="text/javascript">
+	jQuery(function () {
+    	jQuery("#togglePassword").click(function () {
+        	jQuery(this).toggleClass("fa-eye fa-eye-slash");
+            var type = jQuery(this).hasClass("fa-eye-slash") ? "text" : "password";
+            jQuery("#password").attr("type", type);
+        });
```

Comment:
The icon would look more clear when put inside the bars, However i remember @ibacher  commenting about this that we should cater for authorization

Reply:
>  inside the bars,

Hey @sherrif10 
Am sorry to ask but which bars are that 



> commenting about this that we should cater for authorization

Did @ibacher comment here, I am not seeing the comment.  I request you to send me the link here.